support/hololens/ServoApp/ServoControl/Keys.h

```diff
@@ -0,0 +1,130 @@
+using namespace winrt::Windows::System;
+
+namespace winrt {
+std::optional<const char *> KeyToString(Windows::System::VirtualKey key) {
```

Comment:
What happens for letters and digits?

Reply:
Non-printable keys are handled by the `PreviewKeyDown` event.
Printable keys are handled by the `TextUpdating` event.

Reply:
To clarify what's going on here: we either send "raw" key events to servo. For example: `shift` key, then `a` key. Then it would be up to Servo combine those to print `A`, not just `a`. This work is already done by the embedder, so we send what needs to be printed directly.